epoxy-adapter/src/main/java/com/airbnb/epoxy/BaseEpoxyAdapter.java

```diff
@@ -140,6 +138,32 @@ protected void onModelBound(EpoxyViewHolder holder, EpoxyModel<?> model, int pos
 
   }
 
+  @CallSuper
+  @Override
+  public void onAttachedToRecyclerView(RecyclerView recycler) {
+    super.onAttachedToRecyclerView(recycler);
+
+    // set our item animator decorator so we can intercept calls to the item animator
+    // and the user doesn't have to worry about setting the decorator himself
+    if (!(recycler.getItemAnimator() instanceof ChangeAnimationDelegator)) {
```

Comment:
Related thought - if they add their own animator after the adapter is attached it would not be wrappe, but adding it before would wrap it. This seems like confusing behavior, so it might be clearer if we never automatically wrap their custom animator.